lib/modules/platform/bitbucket-server/index.spec.ts

```diff
@@ -247,6 +251,10 @@ describe('modules/platform/bitbucket-server/index', () => {
         });
 
         it('should init', async () => {
+          httpMock
+            .scope(urlHost)
```

Comment:
wrong url

Reply:
Sorry, I am not sure how this is wrong? I logged them and they seemed correct, also the other tests use the same.